cel/options.go

```diff
@@ -68,7 +65,9 @@ func CustomTypeProvider(provider ref.TypeProvider) EnvOption {
 
 // Declarations option extends the declaration set configured in the environment.
 //
-// Note: This option must be specified after ClearBuiltIns if both are used together.
+// Note: Declarations will by default be appended to the pre-existing declaration set cnofigured
+// for the environment. The cel.NewEnv call builds on top of the standard CEL declarations. For
+// a purely custom set of declarations use cel.NewCustomEnv.
```

Comment:
This is the only place in the documentation where you use the namespaced name in the doc comment. Since this function also lives in `cel`, perhaps it is unnecessary?

```suggestion
// for the environment. The NewEnv call builds on top of the standard CEL declarations. For
// a purely custom set of declarations use NewCustomEnv.
```